crates/chain/tests/test_spk_txout_index.rs

```diff
@@ -1,18 +1,21 @@
 use bdk_chain::SpkTxOutIndex;
-use bitcoin::{hashes::hex::FromHex, OutPoint, PackedLockTime, Script, Transaction, TxIn, TxOut};
+use bitcoin::{
+    absolute::{Height, LockTime},
+    OutPoint, ScriptBuf, Transaction, TxIn, TxOut,
+};
 
 #[test]
 fn spk_txout_sent_and_received() {
-    let spk1 = Script::from_hex("001404f1e52ce2bab3423c6a8c63b7cd730d8f12542c").unwrap();
-    let spk2 = Script::from_hex("00142b57404ae14f08c3a0c903feb2af7830605eb00f").unwrap();
+    let spk1 = ScriptBuf::from_hex("001404f1e52ce2bab3423c6a8c63b7cd730d8f12542c").unwrap();
+    let spk2 = ScriptBuf::from_hex("00142b57404ae14f08c3a0c903feb2af7830605eb00f").unwrap();
 
     let mut index = SpkTxOutIndex::default();
     index.insert_spk(0, spk1.clone());
     index.insert_spk(1, spk2.clone());
 
     let tx1 = Transaction {
         version: 0x02,
-        lock_time: PackedLockTime(0),
+        lock_time: LockTime::Blocks(Height::ZERO),
```

Comment:
In e83ea8e1204833049e8d1dd5c6b2be4c981e463d:

There is a `Locktime::ZERO` constant you can use directly.

Also, as a style thing, I find it clearer to import `absolute` and `relative` modules and then to use `absolute::LockTime` everywhere so it's clear what kind of locktime is under consideration.